src/Symfony/Component/ClassLoader/ClassMapGenerator.php

```diff
@@ -134,7 +134,7 @@ private static function findClasses($path)
                     }
 
                     if ($isClassConstant) {
-                        continue;
+                        break;
```

Comment:
But at least I can warn people using my plugin that something might go wrong and they can concentrate on right things instead of long debugging hours.